setup.py

```diff
@@ -108,6 +106,7 @@ def main():
         keywords='box oauth2 sdk',
         license='Apache Software License, Version 2.0, http://www.apache.org/licenses/LICENSE-2.0',
         package_data={'boxsdk': ['py.typed']},
+        python_requires='>=3.8',
```

Comment:
Do we consider this a breaking change? If so, we should include exclamation point in the PR name `!` so the release notes are generated correctly - see https://github.com/box/box-node-sdk/pull/897